org.eclipse.xtext.tests/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,7 @@ Bundle-Localization: plugin
 Bundle-RequiredExecutionEnvironment: JavaSE-11
 Require-Bundle: org.eclipse.xtext;bundle-version="2.35.0",
  org.eclipse.emf.mwe.utils;bundle-version="1.10.0",
- org.eclipse.emf.ecore;bundle-version="2.26.0",
+ org.eclipse.emf.ecore;bundle-version="2.26.0";visibility:=reexport,
```

Comment:
Not sure, this was added when running the workflow. Sadly.

Reply:
Do you think it's bad?